app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagConnection.java

```diff
@@ -112,18 +112,18 @@ public static OkHttpClient getClient() {
     }
 
     public static OkHttpClient createClient() {
-        OkHttpClient client = null;
+        OkHttpClient.Builder b = new OkHttpClient.Builder()
+                .readTimeout(45, TimeUnit.SECONDS);
```

Comment:
I removed `connect` and `write` timeouts because they were equal to the default values. I don't mind getting them back.